resources/calico/config.yaml

```diff
@@ -17,7 +21,7 @@ data:
           "log_level": "info",
           "datastore_type": "kubernetes",
           "nodename": "__KUBERNETES_NODE_NAME__",
-          "mtu": ${network_mtu},
+          "mtu": "__CNI_MTU__",
```

Comment:
I should have guessed this won't be handled sanely.

`json: cannot unmarshal string into Go struct field NetConf.mtu of type int`